Tests/TraceModelTests.swift

```diff
@@ -337,4 +337,55 @@ final class TraceModelTests: XCTestCase {
         XCTAssertEqual(model.spans.count, 5002)
         XCTAssertTrue(model.isComplete)
     }
+    
+    func testTraceSpan_valid() {
```

Comment:
NIT: 
1. Add another test where the start and end time are the same.  
2. Add a test for when the end time is not present

Reply:
Thanks. Updated.